pytest_energy_reporter/util.py

```diff
@@ -0,0 +1,21 @@
+
+def print_table_str(headers=list[str], values=list[list[str]]) -> list[str]:
+    '''Print a table with headers and values as strings. Return a list of strings, each representing a row of the table.'''
+    # Find the maximum width of the columns
+    column_widths = [len(header) for header in headers]
+    for row in values:
+        for i, cell in enumerate(row):
+            column_widths[i] = max(column_widths[i], len(str(cell)))
```

Comment:
I would apply a maximum width for each of the columns. You could truncate the "Test" column for example in the middle. It is not a show-stopper, but for long, describing, test names this will output broken tables.